src/scss/abstracts/_utilities.scss

```diff
@@ -137,51 +137,47 @@
   font-size: $h2-font-size;
   line-height: 1.25;
   font-weight: $font-weight-medium;
+  // https://garyridgway.github.io/clampCalculator/?&minWidth=600&maxWidth=1310&minFont=1.95&maxFont=2.3
+  font-size: clamp(1.95rem, calc(0.7887vw + 1.6542rem), 2.3rem);
```

Comment:
Should we get rid of the other `font-size` declaration?

Reply:
The first `font-size` was there as a fallback for older browsers.  We can probably remove it now?

Reply:
No worries if it is a valid use case. Just noticed it appeared to be a duplicate.